Makefile

```diff
@@ -26,9 +26,8 @@ DOCKERFILE := $(shell bash -c 'source hack/make/.detect-daemon-osarch && echo $$
 DOCKER_GITCOMMIT := $(shell git rev-parse --short HEAD || echo unsupported)
 export DOCKER_GITCOMMIT
 
-# allow overriding the repository and branch that validation scripts are running
-# against these are used in hack/validate/.validate to check what changed in the PR.
-export VALIDATE_REPO
+# allow overriding the branch that validation scripts are running against. These
+# are used in hack/validate/.validate to check what changed in the PR.
 export VALIDATE_BRANCH
 export VALIDATE_ORIGIN_BRANCH
```

Comment:
This was added in https://github.com/moby/moby/pull/38677, but I wonder if it's actually used (@tiborvass ?)